docs2/site/docs/migrations/migration4.md

```diff
@@ -9,6 +9,7 @@
 * Document caching supported via `IDocumentCache` and a default implementation within `DefaultDocumentCache`.
   Within the `GraphQL.Caching` nuget package, a memory-backed implementation is available which is backed by `Microsoft.Extensions.Caching.Memory.IMemoryCache`.
 * `ExecutionOptions.EnableMetrics` is disabled by default
+* Ability to expose user-defined meta-information via introspection. See https://github.com/graphql/graphql-spec/issues/300
```

Comment:
I suggest adding a note that MetaData is not thread safe

Reply:
Also probably need a few more notes about the changes made in this PR...?

Reply:
Done. I write notes after I commit, because it is convenient to watch changes in the GitHub UI.